ouroboros-consensus/src/Ouroboros/Consensus/NodeNetwork.hs

```diff
@@ -527,9 +529,11 @@ consensusNetworkApps kernel ProtocolTracers {..} ProtocolCodecs {..} ProtocolHan
           (chainDbView (getChainDB kernel))
           (getNodeCandidates kernel)
           them $ \varCandidate->
-        runPipelinedPeer
+        runPipelinedPeerWithLimits
           (contramap (TraceLabelPeer them) ptChainSyncTracer)
           pcChainSyncCodec
+          (byteLimitsChainSync (const 0)) -- TODO: Real Bytelimits
```

Comment:
Is there maybe a ticket these new TODOs can refer to?

Reply:
Yes, #1727 will update the comment.